source/_integrations/homematic.markdown

```diff
@@ -485,16 +485,16 @@ template:
   - binary_sensor:
       - name: "Homematic is sending updates"
         state: >-
-          {{ now() - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
+          {{ as_timestamp(now()) - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
```

Comment:
We don't need to calculate using `as_timestamp` anymore, just use `as_datetime`, which can be subtracted from `now()` directly, resulting in a diff.

Additionally, I don't think you can get `last_changed` using `state_attr` actually

Reply:
Good to know, i will update the code to use as_datetime.

Indeed I was not able to get the `last_changed` using `state_attr` and ended up using `states.sensor['xyz].last_changed` which felt like a workaround - so i left it unchanged. Will update to my solution.